music_assistant/server/providers/deezer/__init__.py

```diff
@@ -413,7 +413,7 @@ async def get_audio_stream(
 
         buffer = bytearray()
         async with self.mass.http_session.get(
-            streamdetails.data, headers=headers, timeout=timeout
+            streamdetails.data, headers=headers, timeout=timeoutOh yeah okay
```

Comment:
?

Reply:
Oh whoops

Reply:
Looks like i accidentally pasted stuff all over the place... 😬